measurement/measures/distance.py

```diff
@@ -156,6 +156,20 @@ class Area(MeasureBase):
         ]
     )
 
+    UNITS.update(
```

Comment:
I believe while you are at it, we could clean up how the `UNITS` dictionary is build above and include the values here.
Something like:
```python
UNITS = {
    **{
        f'{AREA_PREFIX}{k}': v **2
        for k, v in Distance.get_units().items()
    },
    'acre': 43560 * (Distance(ft=1).m ** 2),
}
```